ExampleApp iOS/flowforms/Screens/Form/FormView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FormView.swift
+//  FlowForms
+//
+//  Created by Tarek Radovan on 26/10/2022.
+//
+
+import SwiftUI
+import shared
+
+struct FormView: View {
+  
+  @ObservedObject private var formManager: FormManager
+  @State var showPrompt: Bool = false
+  
+  init(formManager: FormManager = FormManager()) {
+    self.formManager = formManager
+  }
+
+  func signUp() {
+    showPrompt(true)
+    DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+      showPrompt(false)
+    }
+  }
+  
+  func showPrompt(_ show: Bool) {
+    withAnimation(.spring(
+      response: Animation.Duration.veryShort,
+      dampingFraction: Animation.Duration.short,
+      blendDuration: Animation.Duration.veryShort
+    )) {
+      showPrompt = show
+    }
+  }
+  
+  var body: some View {
+    NavigationView {
+      VStack {
+        Spacer()
+        VStack(spacing: UI.Padding.medium) {
+          FormModelTextView(
+            title: LocalizedString.FFormView.nameTextfieldTitle,
+            value: formManager.name,
+            errorMessage: formManager.nameErrorMessage
+          )
+          HStack {
+            FormModelTextView(
+              title: LocalizedString.FFormView.emailTextfieldTitle,
+              value: formManager.email,
+              errorMessage: formManager.emailErrorMessage
+            )
+            if formManager.emailVerificationInProgress {
+              ProgressView()
+            }
+          }
+          FormModelTextView(
+            title: LocalizedString.FFormView.passwordTextfieldTitle,
+            secureField: true,
+            value: formManager.password,
+            errorMessage: formManager.passwordErrorMessage
+          )
+          FormModelTextView(
+            title: LocalizedString.FFormView.passwordConfirmationTextfieldTitle,
+            secureField: true,
+            value: formManager.confirmPassword,
+            errorMessage: formManager.confirmedPasswordErrorMessage
+          )
+        }
+        .padding([.leading, .trailing], UI.Padding.large)
+        .frame(alignment: .center)
+        Toggle(isOn: formManager.termsAccepted) {
+          Text(LocalizedString.FFormView.termsAndConditionsText)
+        }
+        .tint(.pink)
+        .padding([.leading, .trailing], UI.Padding.large)
+        Spacer()
+        Button {
+         ///bind to form
```

Comment:
I think the bind is already done via `.disabled` so I believe we don't need this comment anymore